components/cart.component.js

```diff
@@ -0,0 +1,12 @@
+export class CartIconComponent {
+  #page;
+  #cartIcon;
+  constructor(page) {
+    this.#page = page;
```

Comment:
let's simplify it

```suggestion
    #page = page;
```

seems like we can do it

Reply:
...hereinafter

Reply:
unfortunately, js does not support accessing class properties without using this keyword